permits/migrations/0056_get_rid_of_term_travaux.py

```diff
@@ -0,0 +1,64 @@
+            name='works_object_type_choice',
+            field=models.ForeignKey(blank=True, db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='permits.worksobjecttypechoice', verbose_name='objet'),
+        ),
+        migrations.AlterField(
+            model_name='historicalworksobjectpropertyvalue',
+            name='works_object_type_choice',
+            field=models.ForeignKey(blank=True, db_constraint=False, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='+', to='permits.worksobjecttypechoice', verbose_name='objet'),
+        ),
+        migrations.AlterField(
+            model_name='permitactortype',
+            name='works_type',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='works_contact_types', to='permits.workstype', verbose_name='type de demande'),
+        ),
+        migrations.AlterField(
+            model_name='permitrequest',
+            name='works_object_types',
+            field=models.ManyToManyField(related_name='permit_requests', through='permits.WorksObjectTypeChoice', to='permits.WorksObjectType', verbose_name='Objets et types de demandes'),
+        ),
+        migrations.AlterField(
+            model_name='permitrequestamendproperty',
+            name='works_object_types',
+            field=models.ManyToManyField(related_name='amend_properties', to='permits.WorksObjectType', verbose_name='objets'),
+        ),
+        migrations.AlterField(
+            model_name='permitrequestamendpropertyvalue',
+            name='works_object_type_choice',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='amend_properties', to='permits.worksobjecttypechoice', verbose_name='objet'),
+        ),
+        migrations.AlterField(
+            model_name='worksobjectproperty',
+            name='works_object_types',
+            field=models.ManyToManyField(related_name='properties', to='permits.WorksObjectType', verbose_name='objets'),
+        ),
+        migrations.AlterField(
+            model_name='worksobjectpropertyvalue',
+            name='works_object_type_choice',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='properties', to='permits.worksobjecttypechoice', verbose_name='objet'),
+        ),
+        migrations.AlterField(
+            model_name='worksobjecttype',
+            name='works_object',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='works_object_types', to='permits.worksobject', verbose_name='objet'),
+        ),
+        migrations.AlterField(
+            model_name='worksobjecttype',
+            name='works_type',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='works_object_types', to='permits.workstype', verbose_name='type'),
+        ),
+    ]
```

Comment:
please squash migrations

Reply:
Ok, will do it after addressing your comments.

What's the command?

Reply:
As there is no other PR impacting model, juste delete your 3 migrations files and run `makemigrations `again